### DISCUSSION TITLE
Python Supabase unable to import module

### DISCUSSION THREAD

**Discussion by Fxrquaad** [vercel/community#1986] [Help]
### Summary

I've recently implemented a supabae database into my app, after adding supabase into the requirements pushing to github etc. once its deployed im getting the error:

[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': No module named 'supabase'
Traceback (most recent call last):

i've doubled checked my github to make sure it was pushed, no spelling mistakes etc, python 3.9 on both vercel and local machine

**Comment by bassiebal** [comment]
I have the exact same issue unfortunately

**Comment by mubashardev** [comment]
I had the same problem and after searching I didn't find any appropriate solution for anywhere. But I kept trying my own with different scenarios. Finally I was able to fix the issue using `pipenv`.
Make sure you are adding the `supabase` in **requirement.txt** as well as you should run command `pipenv install supabase`, this will install and create a **Pipfile** in your project. And if you're already having a **Pipfile** and it stuck at creating `Pipfile.lock` file then you should delete the `Pipfile.lock` and try the run again `pipenv install supabase`. This will create a fresh `Pipfile.lock` without any problem. The try to push your code to vercel and this will work like a charm. 

For anyhow if it still goes through the same error then go to project settings in vercel dashboard and downgrade the node version and save, and do another code push. This is a bug on vercel sometimes.
